storage_service/locations/migrations/0036_archipelago.py

```diff
@@ -0,0 +1,89 @@
+# Generated by Django 3.2.22 on 2023-10-31 11:45
+import django.db.models.deletion
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("locations", "0035_django32"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="space",
+            name="access_protocol",
+            field=models.CharField(
+                choices=[
+                    ("ARCHIPEL", "Archipelago"),
+                    ("ARKIVUM", "Arkivum"),
+                    ("DV", "Dataverse"),
+                    ("DC", "DuraCloud"),
+                    ("DSPACE", "DSpace via SWORD2 API"),
+                    ("DSPC_RST", "DSpace via REST API"),
+                    ("FEDORA", "FEDORA via SWORD2"),
+                    ("GPG", "GPG encryption on Local Filesystem"),
+                    ("FS", "Local Filesystem"),
+                    ("LOM", "LOCKSS-o-matic"),
+                    ("NFS", "NFS"),
+                    ("REPLICA", "Write-Only Replica Staging on Local Filesystem"),
+                    ("PIPE_FS", "Pipeline Local Filesystem"),
+                    ("RCLONE", "RClone"),
+                    ("SWIFT", "Swift"),
+                    ("S3", "S3"),
+                ],
+                help_text="How the space can be accessed.",
+                max_length=8,
+                verbose_name="Access protocol",
+            ),
+        ),
+        migrations.CreateModel(
+            name="Archipelago",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "archipelago_url",
+                    models.URLField(
+                        help_text="Archipelago URL",
+                        max_length=256,
+                        verbose_name="Archipelago URL",
+                    ),
+                ),
+                (
+                    "archipelago_user",
+                    models.CharField(
+                        help_text="Archiipelago username for authentication",
```

Comment:
```suggestion
                        help_text="Archipelago username for authentication",
```
Spelling needs to be changed.